src/loop/typer_intf.ml

```diff
@@ -268,11 +279,11 @@ module type S = sig
   val print : Format.formatter -> typechecked stmt -> unit
   (** Printing funciton for typechecked statements. *)
 
-  val check : state -> Dolmen.Std.Statement.t -> state * typechecked stmt
+  val check : state -> Dolmen.Std.Statement.t -> state * typechecked stmt list
```

Comment:
Maybe would be worth documenting why this returns a list and its order? I think implicits are first?

Reply:
doc added